apps/coreneuron.cpp

```diff
@@ -27,7 +27,10 @@ THE POSSIBILITY OF SUCH DAMAGE.
 */
 
 #include <coreneuron/enginemech.h>
+#include "coreneuron/nrniv/profiler_interface.h"
 
 int main(int argc, char** argv) {
-    return solve_core(argc, argv);
+    coreneuron::Instrumentor::init_profile();
+    solve_core(argc, argv);
```

Comment:
the `main` function should return what the `solve_core` function returns.

Reply:
Should be solved by 45a48f64aa4640131ac94ce21eec8f1919bd5559

Reply:
Thank you!